### ISSUE TITLE
Left/Cross join with function

### ISSUE THREAD

**Issue by AbuSM** [kayak/pypika#550]
Want to realize ```left join "bill".get_account_funds("a"."unique_id") balance on true``` using pypika. Database is PostgreSQL. The whole query works, but no idea to make it in pypika

**Comment by tandonraghavs** [comment]
This also includes cases like - `select prd from a CROSS JOIN UNNEST(array) as pr(prd)`
Can someone please explain, how to write such queries?

**Comment by pg** [comment]
Yes, is there a way to support the `CROSS JOIN UNNEST(array) as pr(prd)` portion?
